agg-ast/schema_derivation/src/match_schema_derivation.rs

```diff
@@ -2123,7 +2149,7 @@ impl MatchConstrainSchema for Expression {
                 TaggedOperator::DateDiff(d) => match_derive_date_diff(d, state)?,
                 TaggedOperator::DateSubtract(d) => match_derive_date_subtract(d, state)?,
                 TaggedOperator::DateTrunc(d) => match_derive_date_trunc(d, state)?,
-                _ => todo!(),
+                _ => {}
```

Comment:
did you consider throwing an error here? I suppose for match its probably better default behavior to do nothing then error, if we eg add a new op but forget to add it to match, but curious if you considered it any deeper.

Reply:
Yes, the idea is that we are conservative here.